### PR TITLE
ADR: Disable group chats for beta

### PR DESCRIPTION
**Pull request by oskarth**

### CODE REVIEW

**Comment by rasom** [comment]
apk uploaded to https://i.diawi.com/GuRSBN for branch PR-4244

<blockquote><div><strong><a href="https://i.diawi.com/GuRSBN">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Comment by rasom** [comment]
ipa uploaded to https://i.diawi.com/2JVzgi for branch PR-4244

<blockquote><div><strong><a href="https://i.diawi.com/2JVzgi">Install - Diawi - Development and In-house Apps Wireless Installation</a></strong></div></blockquote>

**Review by chadyj — APPROVED** [review]

File: doc/decisions/0004-disable-group-chats.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+We want to launch our beta in Q2. This requires us to keep a tight scope and stay on the critical path.
+
+Group chat is not as vital as 1-1 chat for the chat experience. It is also
+likely not as useful for an MVP as public chat is, since this gives people a
+chance to talk to strangers and it is open by default.
+
+Whether group chats are in scope or not has been a recurring discussing and a
+lot of effort has been wasted related to this.
+
+There are currently a lot of outstanding regressions related to group chat: https://github.com/status-im/status-react/labels/group-chat Additionally, since group chats are private the encryption and security of those are generally harder than 1-1 and public chat.
```
Comment (pedropombeiro) [inline doc/decisions/0004-disable-group-chats.md:23]: nit: missing dot before `Additionally`

**Review by pedropombeiro — APPROVED** [review]
LGTM

**Pull request merged**